ext/hal/silabs/gecko/Kconfig

```diff
@@ -9,3 +9,8 @@ config HAS_SILABS_GECKO
 	bool
 	select HAS_CMSIS
 	depends on SOC_FAMILY_EXX32
+
+config HAS_GECKO_DCDC
```

Comment:
Not all projects will want to enable internal DC/DC converter. It has a limited power output and some applications will require an external one. This should be a visible option which is configurable and can be enabled by the board configuration file.

Let's rename it to 'GECKO_DCDC' or maybe 'SOC_GECKO_DCDC'.

Reply:
I have added it to the board as a default config instead for it there